src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -300,14 +298,14 @@ static String guessModByFileName(String fileName) {
       return CompletableFuture.allOf(featuredModFuture, mapBeanFuture).thenApply(ignoredVoid  -> {
         Optional<MapBean> mapBean = mapBeanFuture.join();
         if (!mapBean.isPresent()) {
-          throw new CompletionException(new FileNotFoundException());
+          logger.warn("Could not find map for replay file '{}'", replayFile);
```

Comment:
We certainly dont wanna throw here if we do then the whole loacl replay vault breaks, also there are legimit cases where a replay has no map. One is coop the maps are part of the game mod and hence are not found locally nor on the server